root/admin/attributes/form.tt

```diff
@@ -22,14 +21,12 @@
         [% form_row_text(r, 'name', add_colon(l('Name'))) %]
         [% form_row_text(r, 'iso_code', add_colon(l('ISO code'))) %]
         [% form_row_text(r, 'iso_number', add_colon(l('ISO number'))) %]
-        [% form_row_text(r, 'frequency', add_colon(l('Frequency'))) %]
+        [% form_row_select(r, 'frequency', add_colon(l('Frequency'))) %]
         <p>
             [% l('Frequency notes:') %]
             <ul>
-                <li>[% l('4: Shown in the commonly used section') %]</li>
-                <li>[% l('3: Shown in the "other" section, used for scripts which are likely to get some usage') %]</li>
-                <li>[% l('2: Shown in the "other" section, used for scripts in Unicode which are unlikely to be used') %]</li>
-                <li>[% l('1: Hidden, used for scripts not in Unicode') %]</li>
+                <li>[% l('Both Other and Uncommon are shown in the "other" section, but Uncommon should be used for scripts in Unicode which are unlikely to be used.') %]</li>
```

Comment:
You set `OTHER` to 2 in constants but here document to use `UNCOMMON` (set to 3 in constants) instead for scripts unlikely to be used.

(Scripts of frequency 2 and 3 are mixed together in “Other” option group anyway.)

Reply:
Whoops. Put it upside down in constants. Fixed.